ibtk/include/ibtk/FEDataManager.h

```diff
@@ -284,6 +334,23 @@ class FEData : public SAMRAI::tbox::Serializable
  * </ol>
  * The default value is <code>node_outside_error</code>.
  *
+ * <code>subdomain_ids_on_levels</code>: a database correlating libMesh subdomain
+ * IDs to patch levels. A possible value for this is
+ * @code
+ * subdomain_ids_on_levels
+ * {
+ *   level_1 = 4
+ *   level_3 = 10, 12, 14
+ *   level_5 = 1000, 1003, 1006, 1009
+ * }
+ * @endcode
+ * This particular input will associate all elements with subdomain id 4 with
+ * patch level 1, all elements with subdomain ids 10, 12, or 14 with patch level
+ * 3, etc. All unspecified subdomain ids will be associated with the finest
+ * patch level. All inputs in this database for levels finer than the finest
+ * level are ignored (e.g., if the maximum patch level number is 4, then the
+ * values given in the example for level 5 ultimately end up on level 4).
```

Comment:
The documentation is ahead of what is actually implemented.

Reply:
👍